README.md

```diff
@@ -8,6 +8,7 @@ This application provides a reverse proxy with TLS termination, supporting confi
 
 - Client-side TLS termination with confidentialVM attestation verification.
 - Server-side TLS termination with confidentialVM attestation verification.
+- Mutual attestations between client and server.
```

Comment:
Will introduce a conflict with https://github.com/flashbots/cvm-reverse-proxy/pull/8, maybe remove (the other one introduces this in some more detail)